### PR TITLE
Remove dso file

### PR DESCRIPTION
**Pull request by simoncross747**

### CODE REVIEW

File: tools/config/azure-provider-config.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
-{"devtest": {
-  "resource_group_name": "webops",
-  "storage_account_name": "nomsstudiowebops",
+{
+  "comment": "This file's purpose is to allow you to override values in config.tf.json, which is regenerated when init.py or diginit are invoked.",
```
Comment (glenjamin) [inline tools/config/azure-provider-config.json]: Does this override values if you change it & run `diginit`, or only affect default values?

As long as we still have the ability to vary versions per folder we're all good.

Comment (simoncross747) [reply tools/config/azure-provider-config.json]: It checks for  azure-provider-config.json locally and if it doesn't exist it copies the default config allowing us to vary the Terraform/Azure versions.

Comment (glenjamin) [reply tools/config/azure-provider-config.json]: Then I'm happy :)

**Comment by glenjamin** [comment]
@simoncross747 are there a bunch of `dso.json` files we need to delete now?

**Pull request merged**